### PR TITLE
Update installer

### LINKED ISSUE #21: Update installer
#### Summary
The current installer is based on the old Golang node, combined with Rusk. To prepare the ITN installer for ITN2, we need to update the scripts.

#### Possible solution design or implementation
The installer no longer needs to use the `dusk` and `rusk-recovery-state` binary. We should remove these and replace the `rusk` binary with the latest version.

Given the blockchain is encapsulated into a single binary, the way services are structure in the installer also needs to change.

### PR DESCRIPTION
**Pull request by HDauven**
Resolves #21

### CODE REVIEW

File: services/rusk.service [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+Environment="RUSK_PROFILE_PATH=/opt/dusk/.rusk"
 
 User=dusk
 WorkingDirectory=/opt/dusk
 
-ExecStartPre=!/bin/bash -c '/opt/dusk/bin/rusk-recovery-state >> /var/log/rusk_recovery.log'
-ExecStartPre=!/bin/bash -c 'chown -R dusk $RUSK_PROFILE_PATH'
+ExecStartPre=!/bin/bash -c '/opt/dusk/bin/rusk recovery-keys >> /var/log/rusk_recovery.log'
+ExecStartPre=!/bin/bash -c '/opt/dusk/bin/rusk recovery-state >> /var/log/rusk_recovery.log'
 ExecStartPre=!/bin/bash -c '/opt/dusk/bin/check_consensuskeys.sh'
```
Comment (herr-seppia) [inline services/rusk.service:17]: You should update check_consensus_keys.sh to check the proper path

**Review by herr-seppia — CHANGES_REQUESTED** [review]

File: conf/wallet.toml
[the remark this answers is not in the document]
Comment (HDauven) [reply conf/wallet.toml:1]: Are you sure? I think setting this as the default for the droplet wallet makes sense, no?

File: itn-installer.sh [hunk 2]
```diff
@@ -47,24 +44,19 @@ mv -n /opt/dusk/installer/services/* /opt/dusk/services/
...
 
-mkdir -p /opt/dusk/data/.rusk
-mkdir -p /opt/dusk/data/chain
-
 echo "Downloading verifier keys"
-curl -so /opt/dusk/installer/vd-keys.zip -L "$VERIFIER_KEYS_URL"
-unzip -d /opt/dusk/data/ -o /opt/dusk/installer/vd-keys.zip
-
+curl -so /opt/dusk/installer/rusk-vd-keys.zip -L "$VERIFIER_KEYS_URL"
+unzip -d /opt/dusk/.rusk/ -o /opt/dusk/installer/rusk-vd-keys.zip
```
[the remark this answers is not in the document]
Comment (HDauven) [reply itn-installer.sh]: -d doesn't exist AFAIK, used dusk:dusk instead

File: services/rusk.service [hunk 3]
```diff
@@ -1,33 +1,36 @@
...
 
 Environment="RUST_BACKTRACE=full"
-Environment="RUSK_PROFILE_PATH=/opt/dusk/data"
+Environment="RUSK_PROFILE_PATH=/opt/dusk/rusk"
 
 User=dusk
 WorkingDirectory=/opt/dusk
 
-ExecStartPre=!/bin/bash -c '/opt/dusk/bin/rusk-recovery-state >> /var/log/rusk_recovery.log'
-ExecStartPre=!/bin/bash -c 'chown -R dusk $RUSK_PROFILE_PATH'
```
Comment (HDauven) [inline services/rusk.service:17]: @herr-seppia shame call out

**Review by herr-seppia — APPROVED** [review]

**Pull request merged**